### ISSUE TITLE
Error in running extraction from module Logcat: 'utf-8' codec can't decode byte 0xe8 in position 6066334: invalid continuation byte                      

### ISSUE THREAD

**Issue by Douglas-cyber** [mvt-project/mvt#191]
```
ERROR    [mvt.android.modules.adb.logcat] Error in running extraction from module Logcat: 'utf-8' codec can't decode byte 0xe8 in position 6066334: invalid continuation 
                  byte                                                                                                                                                            
                  Traceback (most recent call last):                                                                                                                              
                    File "$HOME/.local/lib/python3.9/site-packages/mvt/common/module.py", line 153, in run_module                                                          
                      module.run()                                                                                                                                                
                    File "$HOME/.local/lib/python3.9/site-packages/mvt/android/modules/adb/logcat.py", line 27, in run                                                     
                      output = self._adb_command("logcat -d")                                                                                                                     
                    File "$HOME/.local/lib/python3.9/site-packages/mvt/android/modules/adb/base.py", line 111, in _adb_command                                             
                      return self.device.shell(command)                                                                                                                           
                    File "$HOME/.local/lib/python3.9/site-packages/adb_shell/adb_device.py", line 792, in shell                                                            
                      return self._service(b'shell', command.encode('utf8'), transport_timeout_s, read_timeout_s, timeout_s, decode)                                              
                    File "$HOME/.local/lib/python3.9/site-packages/adb_shell/adb_device.py", line 681, in _service                                                         
                      return b''.join(self._streaming_command(service, command, transport_timeout_s, read_timeout_s, timeout_s)).decode('utf8')                                   
                  UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe8 in position 6066334: invalid continuation byte        
```

**Comment by stt** [comment]
Slightly different message but probably same issue here

    ERROR    [mvt.android.modules.adb.logcat] Error in running extraction from module Logcat:
             'utf-8' codec can't decode byte 0xc0 in position 1732541: invalid start byte
             Traceback (most recent call last):
               File "/Users/s/.pyenv/versions/3.9.6/lib/python3.9/site-packages/mvt/common/module.py", line 153, in run_module                                        
                 module.run()

**Comment by cracuxi** [comment]
Same issue:
`         INFO     [mvt.android.modules.adb.logcat] Running module Logcat...                                                             
19:25:21 ERROR    [mvt.android.modules.adb.logcat] Error in running extraction from module Logcat: 'utf-8' codec can't decode byte 0x9d 
                  in position 4763658: invalid start byte                                                                               
                  Traceback (most recent call last):                                                                                    
                    File "/usr/src/mvt/mvt/common/module.py", line 153, in run_module                                                   
                      module.run()                                                                                                      
                    File "/usr/src/mvt/mvt/android/modules/adb/logcat.py", line 27, in run                                              
                      output = self._adb_command("logcat -d")                                                                           
                    File "/usr/src/mvt/mvt/android/modules/adb/base.py", line 114, in _adb_command                                      
                      return self.device.shell(command)                                                                                 
                    File "/home/tester/.local/lib/python3.9/site-packages/adb_shell/adb_device.py", line 792, in shell                  
                      return self._service(b'shell', command.encode('utf8'), transport_timeout_s, read_timeout_s, timeout_s, decode)    
                    File "/home/tester/.local/lib/python3.9/site-packages/adb_shell/adb_device.py", line 681, in _service               
                      return b''.join(self._streaming_command(service, command, transport_timeout_s, read_timeout_s,                    
                  timeout_s)).decode('utf8')                                                                                            
                  UnicodeDecodeError: 'utf-8' codec can't decode byte 0x9d in position 4763658: invalid start byte  `
Phone is a Xiaomi, therefore, likely to be a Chinese character

**Comment by botherder** [comment]
Might be good to flag to @JeffLIrion

**Comment by JeffLIrion** [comment]
This is fixed in the latest `adb-shell` release (0.4.1).

**Comment by botherder** [comment]
Thanks!
